pkg/products/amqonline/reconciler_test.go

```diff
@@ -184,6 +186,21 @@ func setupRecorder() record.EventRecorder {
 }
 
 func TestReconcile_reconcileAuthServices(t *testing.T) {
+
+	postgres := &crov1.Postgres{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test-postgres-",
+			Namespace: "test-namespace",
+		},
+		Spec: crov1.PostgresSpec{},
+		Status: crov1.PostgresStatus{
+			Phase: crotypes.PhaseComplete,
+			SecretRef: &crotypes.SecretRef{
+				Name:      "test-postgres-",
+				Namespace: "test-postgres-namespace",
```

Comment:
```suggestion
		ObjectMeta: metav1.ObjectMeta{
			Name:      "standard-authservice-postgresql",
			Namespace: "test-namespace",
		},
		Spec: crov1.PostgresSpec{},
		Status: crov1.PostgresStatus{
			Phase: crotypes.PhaseComplete,
			SecretRef: &crotypes.SecretRef{
				Name:      "standard-authservice-postgresql",
				Namespace: "test-namespace",
```
I think unit tests are failing with these changes :thinking: